pkg/controller/mizar/mizar-arktos-network-controller.go

```diff
@@ -174,10 +197,44 @@ func (c *MizarArktosNetworkController) syncNetwork(eventKeyWithType KeyWithEvent
 }
 
 func (c *MizarArktosNetworkController) processNetworkCreation(network *v1.Network, eventKeyWithType KeyWithEventType) error {
+	//Find out the paths of default template to create vpc and subnet
+	vpc := network.Spec.VPCID
+	subnet := vpc + subnetSuffix
+
+	currentDir, err := os.Getwd()
+	if err != nil {
+		klog.Errorf("Mizar-Arktos-Network-controller: Get current directory (%s) in error (%v).", currentDir, err)
+		return err
+	}
+
+	if !strings.HasSuffix(currentDir, arktosName) {
+		klog.Errorf("Mizar-Arktos-Network-controller: Current directory (%s) is not in Arktos Home directory with error (%v).", currentDir, err)
+		return err
+	}
+
+	templateDir := currentDir + homeSubPath
+
+	vpcDefaultTemplatePath := templateDir + "default_mizar_network_vpc_template.json"
+	subnetDefaultTemplatePath := templateDir + "default_mizar_network_subnet_template.json"
+
+	klog.V(4).Infof("Mizar-Arktos-Network-controller - vpcPath: (%s) + subnetPath: (%s)", vpcDefaultTemplatePath, subnetDefaultTemplatePath)
```

Comment:
Log will automatically have file name "mizar-arktos-network-controller.go", no need to have the prefix in all the log.

Reply:
I will remove all prefix in all the log.

Reply:
Done.